test/tattr.c

```diff
@@ -4009,7 +4009,16 @@ test_attr_big(hid_t fcpl, hid_t fapl)
     /* Create attribute */
     u = 2;
     snprintf(attrname, sizeof(attrname), "attr %02u", u);
-    attr = H5Acreate2(dataset, attrname, H5T_NATIVE_UINT, big_sid, H5P_DEFAULT, H5P_DEFAULT);
+
+    if (vol_is_native && low != H5F_LIBVER_LATEST) {
```

Comment:
This replicates the logic below and suppresses an error stack output from an expected failure